packages/cli/src/commands/link/index.ts

```diff
@@ -68,17 +80,36 @@ export default async function main(client: Client) {
     argv['--yes'] = argv['--confirm'];
   }
 
-  const cwd = argv._[1] || process.cwd();
-
-  const link = await ensureLink('link', client, cwd, {
-    autoConfirm: !!argv['--yes'],
-    forceDelete: true,
-    projectName: argv['--project'],
-    successEmoji: 'success',
-  });
+  const yes = !!argv['--yes'];
+
+  let cwd = argv._[1];
+  if (cwd) {
+    client.output.warn(
+      `The ${cmd('vc link <directory>')} syntax is deprecated, please use ${cmd(
+        `vc link --cwd ${cwd}`
+      )} instead`
+    );
+  } else {
+    cwd = process.cwd();
+  }
 
-  if (typeof link === 'number') {
-    return link;
+  if (argv['--repo']) {
+    client.output.warn(
+      `The ${cmd('--repo')} flag is in alpha, please report issues`
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏Can we note here (and wherever else we say `(alpha)`) that this linking style will only work with specific commands?